lighthouse-core/audits/csp-xss.js

```diff
@@ -74,14 +76,14 @@ class CspXss extends Audit {
 
   /**
    * @param {Finding} finding
-   * @param {string=} icon
+   * @param {(string|LH.IcuMessage)=} severity
```

Comment:
do we actually have strings left? seems like we might be able to enforce IcuMessage